test/test-webdriver-plus.ts

```diff
@@ -51,6 +61,21 @@ describe('webdriver-plus', () => {
       assert.deepEqual(elemsMapped, ['World!', 'OK']);
     });
 
+    it('should wait for match with driver.findWait()', async function() {
+      // Add component after 500ms.
+      setTimeout(() => driver.executeScript(addDom, 'waitForIt'), 500);
```

Comment:
I think you could make all these tests a bunch faster and more precise. The idea is you start the wait immediately. At the same time, in a separate "thread" of sorts, wait a tiny bit, like 10ms, then check that the element is absent, then add the element. That would ensure that the element really was absent when the wait started, so that the wait really works. Something like this
```
async function add() {
   await new Promise((r) => setTimeout(r, 10));  // poor man's delay(), since i don't think we have one.
   assert.equal(await driver.find('#waitForIt').isPresent(), false);
   driver.executeScript(addDom, 'waitForIt');
}
assert.equal((await Promise.all([driver.findWait(1, '#waitForIt').getText(), add]))[0], 'Foo');
```